src/main/kotlin/Decorator/littlekai/base/IngredientDecorator.kt

```diff
@@ -1,7 +1,7 @@
 package Decorator.littlekai.base
 
 abstract class IngredientDecorator (private val noodles: Noodles) : Noodles by noodles {
-  protected abstract val COST : Double
+  abstract val COST : Double
```

Comment:
The code was good. I changed it because I couln't test it without hardcoding. But, normally, variables should be private or protected. Isn't it?